src/store/saga/UserInfoForm/UserInfoForm.ts

```diff
@@ -0,0 +1,31 @@
+import { ADD_USER_INFO } from "store/types";
+import { call, takeLatest, put } from "redux-saga/effects";
+import axios from "axios";
+import { addApiDataFromRedux } from "store/actions/addApiDataFromRedux";
+import { setReduxKey } from "store/actions/App";
+interface ResponseGenerator {
+  data: any;
+}
+let some: any = [];
+const apiCall = (): Promise<ResponseGenerator> => {
+  return axios
+    .get("https://api.publicapis.org/entries")
+    .then((response) => response);
+};
+function* sagaApiData() {
+  try {
+    yield put(setReduxKey("loaderKey", true));
+    const res: ResponseGenerator = yield call(apiCall);
+    for (let i = 0; i < 50; i++) {
+      some.push(res.data.entries[i]);
+    }
+    yield put(addApiDataFromRedux(some));
+    yield put(setReduxKey("loaderKey", false));
+  } catch (error) {
+    console.log("error", error);
+  }
```

Comment:
you should have use Toaster or alertbox to show error

Reply:
done

Reply:
react-toastify applied to show error messages